src/App.template.ts

```diff
@@ -1,7 +1,7 @@
 import type { TemplateMap } from '@muban/template';
 import { html } from '@muban/template';
-import { layoutCustomTemplate } from './components/layouts/CustomLayout.template';
-import { layoutDefaultTemplate } from './components/layouts/DefaultLayout.template';
+import { layoutCustomTemplate } from './components/layouts/custom/CustomLayout.template';
```

Comment:
```suggestion
import { customLayoutTemplate } from './components/layouts/custom/CustomLayout.template';
```
We should also rename the function as it is not following our Coding Standard